ethportal-api/src/types/content_key.rs

```diff
@@ -177,19 +193,29 @@ impl OverlayContentKey for HistoryContentKey {
         sha256.finalize().into()
     }
 
-    fn bytes(&self) -> [u8; 33] {
-        let mut bytes = [0u8; 33];
-        let (selector, hash) = match self {
-            HistoryContentKey::BlockHeader(k) => (0x00, k.block_hash),
-            HistoryContentKey::BlockBody(k) => (0x01, k.block_hash),
-            HistoryContentKey::BlockReceipts(k) => (0x02, k.block_hash),
-            HistoryContentKey::EpochAccumulator(k) => (0x03, k.epoch_hash.0),
-        };
-        for (i, byte) in hash.into_iter().enumerate() {
-            bytes[i] = byte
+    fn bytes(&self) -> Vec<u8> {
```

Comment:
@perama-v Had to change this method to return `Vec<u8>` instead of an array because we want to handle also the case for an `Unknown` content key, which length is not fixed.

Reply:
Good call